tests/resources/Nimbus-Util.robot

```diff
@@ -53,14 +53,15 @@ Deploy Nimbus ESXi Server
     Wait Until Keyword Succeeds  2 min  30 sec  Login  ${user}  ${password}
 
     :FOR  ${IDX}  IN RANGE  1  5
-    \   ${out}=  Execute Command  ${NIMBUS_LOCATION} nimbus-esxdeploy ${name} --disk=48000000 --ssd=24000000 --memory=8192 --lease=0.25 --nics 2 ${version}
+    \   ${out}=  Execute Command  ${NIMBUS_LOCATION} nimbus-esxdeploy ${name} --pxe-base-dir=/dbc/sc-dbc1218/yifengx/pxeboot/${user} --disk=48000000 --ssd=24000000 --memory=8192 --lease=0.25 --nics 2 ${version}
```

Comment:
It doesn't seem like we should associate these deployments with a non-automation account.